kafka/hbase-kafka-proxy/src/main/appended-resources/META-INF/LICENSE.txt

```diff
@@ -0,0 +1,3610 @@
+Software or previous Modification; or
+
+C. Any new file that is contributed or otherwise made
+available under the terms of this License.
+
+1.10. Original Software means the Source Code and Executable
+form of computer software code that is originally
+released under this License.
+
+1.11. Patent Claims means any patent claim(s), now owned or
+ hereafter acquired, including without limitation, method,
+ process, and apparatus claims, in any patent Licensable
+ by grantor.
+
+1.12. Source Code means (a) the common form of computer
+software code in which modifications are made and (b)
+associated documentation included in or with such code.
+
+1.13. You (or Your) means an individual or a legal entity exercising
+rights under, and complying with all of the terms of, this License.
+For legal entities, You includes any entity which controls,
+is controlled by, or is under common control with You.
+For purposes of this definition, control means (a)�the power,
+ direct or indirect, to cause the direction or management of such entity,
+ whether by contract or otherwise, or (b)�ownership of more than
+ fifty percent (50%) of the outstanding shares or beneficial
+ ownership of such entity.
+
+2. License Grants.
+
+2.1. The Initial Developer Grant.
+Conditioned upon Your compliance with Section 3.1 below and
+subject to third party intellectual property claims, the
+Initial Developer hereby grants You a world-wide, royalty-free,
+non-exclusive license:
+(a) under intellectual property rights (other than patent or trademark)
+Licensable by Initial Developer, to use, reproduce, modify,
+display, perform, sublicense and distribute the Original Software
+(or portions thereof), with or without Modifications, and/or
+as part of a Larger Work; and
+(b) under Patent Claims infringed by the making, using or selling
+ of Original Software, to make, have made, use, practice, sell,
+ and offer for sale, and/or otherwise dispose of the
+ Original Software (or portions thereof).
+(c) The licenses granted in Sections�2.1(a) and (b) are effective
+on the date Initial Developer first distributes or otherwise makes
+the Original Software available to a third party under the terms of this License.
+(d) Notwithstanding Section�2.1(b) above, no patent license is granted:
+(1)�for code that You delete from the Original Software, or
+(2)�for infringements caused by: (i)�the modification of the Original Software,
+or (ii)�the combination of the Original Software with other software or devices.
+
+2.2. Contributor Grant.
+Conditioned upon Your compliance with Section 3.1 below and subject to
+third party intellectual property claims, each Contributor hereby grants
+You a world-wide, royalty-free, non-exclusive license:
+(a) under intellectual property rights (other than patent or trademark)
+ Licensable by Contributor to use, reproduce, modify, display, perform,
+ sublicense and distribute the Modifications created by such Contributor
+ (or portions thereof), eith
```

Comment:
similarly to earlier, here we should say we're choosing the ALv2 license. as an asf project we can't distribute under LGPL.